src/Main.cpp

```diff
@@ -26,14 +28,36 @@ int main()
             sf::Event event;
             while (window.pollEvent(event))
             {
-                if (event.type == sf::Event::Closed)
+              if (event.type == sf::Event::Closed)
                     window.close();
+               
+     
+            switch (event.type)
+            paused = !paused;
+            if (event.type == sf::Event::KeyReleased)
+            {
+                if (event.key.code == sf::Keyboard::Escape);
```

Comment:
This is the if statement that checks exactly what you need - whether the Escape button was pressed (released really). The main logic of switching paused state should be here.
However, you're ending it with a ';' so basically it does nothing :D Add the curly braces and put the ```paused != paused``` here

Reply:
I'm have a problem here because VS is telling me to add ";". Why
![what_why](https://user-images.githubusercontent.com/67471471/171019394-0792642b-6a52-4d6b-9075-23a2bb5fe270.PNG)
?

Reply:
remember that the compiler expects the ; at the end of a statement. That's why it marks error with "if", because it was expecting ; but got some 'weird' if :) So usually you have to look at the line before with this type of error

Reply:
I tried again, but the VS still tells me to add ";" after "paused = !paused"
![;_problem_number_two](https://user-images.githubusercontent.com/67471471/171038030-8fac41f3-3a15-42f8-a40f-32961f97a158.PNG)